src/components/date/date-picker.spec.js

```diff
@@ -225,22 +217,37 @@ describe("StyledDayPicker", () => {
   describe("i18n", () => {
     describe("translation", () => {
       it("renders properly", () => {
-        const wrapper = render(
+        const wrapper = renderI18n(
           {
             inputDate: firstDate,
             selectedDate: new Date("2019-04-01"),
+            locale: {
+              locale: () => "fr-FR",
+            },
           },
-          mount,
-          {
-            wrappingComponent,
-          }
+          mount
         );
         expect(wrapper.find(DayPicker).props().locale).toBe("fr-FR");
+        wrapper.setProps({
+          locale: {
+            locale: () => "en-GB",
+          },
+        });
+        expect(wrapper.find(DayPicker).props().locale).toBe("en-GB");
       });
     });
   });
 });
 
+function renderI18n({ locale, ...props }, renderer = shallow, params) {
```

Comment:
do you need to have the `renderer = shallow` here at all? It could just be 

```
  return mount (...
```

Reply:
good point, changed, thanks!